app/components-react/windows/settings/Multistreaming.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Services } from '../../service-provider';
+import { $t } from '../../../services/i18n';
+import { ObsSettingsSection } from './ObsSettings';
+import Translate from 'components-react/shared/Translate';
+import * as remote from '@electron/remote';
+
+export function MultistreamingSettings() {
+  const { UserService, MagicLinkService } = Services;
+
+  async function upgradeToPrime() {
+    const link = await MagicLinkService.getDashboardMagicLink('prime-marketing', 'slobs-ui-themes');
+    remote.shell.openExternal(link);
+  }
+
+  const shouldShowPrime = UserService.views.isLoggedIn && !UserService.views.isPrime;
```

Comment:
This should be accessed with `useVuex` to make sure it's reactive.  `isLoggedIn` and `isPrime` are things that are unlikely to change while this window is open, but still technically can, so best to hook it up to be reactive.

Reply:
Sounds good!